examples/lira_attack_example.py

```diff
@@ -129,7 +135,9 @@
     "python -m aisdc.attacks.likelihood_attack run-attack "
     "--json-file config.json "
     "--report-name example_lira_report "
-    "--n-shadow-models 100"
+    "--n-shadow-models 100 "
+    # "--shadow-models-fail-fast True "
```

Comment:
why not have two examples in the script? One with, and one without these options?

Reply:
In the script just added two examples as suggested.